source/_integrations/matter.markdown

```diff
@@ -23,182 +23,241 @@ ha_platforms:
-2) The Eve device runs on HomeKit by default, you will need an iOS device to set it up out of the box.
-3) The Eve device uses Thread for communication, therefore you will need to have a Thread Border Router configured in your network setup, such as the Apple TV 4K (2021/2022), Homepod Mini, or Homepod V2.
-4) You need to join the [Eve Beta program here](https://www.evehome.com/en/meet-matter) and wait for instructions per email.
-5) Update the firmware of your Eve device using the Eve beta app to Matter.
-6) During the update process the Eve app will create a new QR code for you to save somewhere safe. This QR code is required when you want to add the device to a Matter controller. The normal QR code attached to the device will no longer work!
-7) During the upgrade process, the Eve app will join the device to Apple Home using Matter.
-8) Confirm that the device is working properly within Apple Home.
-9) Follow our instructions above to share the device from Apple Home to Home Assistant.
+- If you see a Matter logo on the box, the device runs Matter already and you can add it to HA immediately.
+- If there is a Thread logo, you need to install the Matter firmware using the Eve app.
+- No Matter logo and no Thread logo on the box? The device is not Matter compatible.
+- The energy metering feature of the Eve plug will not work in Home Assistant (Apple only feature).
+- Eve has just released official Matter support so ignore any documentation about the beta program.
 
-Once the initial firmware upgrade to Matter is complete, the device can also be paired to non-Apple based Thread networks, like Google Home or later Home Assistant's own Thread implementation based on OTBR but you do need an Apple ecosystem running (iOS phone + Border router) for the first time setup.
+[Eve Energy on Amazon](https://amzn.to/3YuO62P)
+[Eve Door & Window on Amazon](https://amzn.to/3RIU6ml)
+[Eve Motion on Amazon](https://amzn.to/3jDujiP)
 
-- [Eve Energy on Amazon](https://amzn.to/3YuO62P)
-- [Eve Door & Window on Amazon](https://amzn.to/3RIU6ml)
-- [Eve Motion on Amazon](https://amzn.to/3jDujiP)
+### Nanoleaf Matter bulbs and Lightstrips
+
+- Although the products work great once commissioned, multiple users have reported that commissioning them can be a bit difficult and requires some patience and multiple resets or optimizations to your home network.
+- Check the [Nanoleaf Matter infopage](https://nanoleaf.me/en-EU/integration/matter/) for all supported products and instructions.
+
+### Philips Hue (V2) Bridge
+
+The Philips Hue V2 bridge supports Matter since a recent update (the beta program closed, it is now officially available). You can enable Matter support from the Hue app after which you can commission it to Home Assistant and other fabrics.
+
+- It does not make a lot of sense to bind the Hue bridge to Home Assistant because you will loose functionality over the default Hue integration in Home Assistant, such as button press events and (dynamic) scenes.
```

Comment:
```suggestion
- Binding the Hue bridge to Home Assistant does not make sense because you will lose functionality over the default Hue integration in Home Assistant, such as button press events and (dynamic) scenes.
```